docs/source/about/execution_model.rst

```diff
@@ -0,0 +1,28 @@
+Execution model
+===============
+
+Since the GraphQL compiler can target multiple different query
+languages, each with its own behaviors and limitations, the execution
+model must also be defined as a function of the compilation target
+language. While we strive to minimize the differences between
+compilation targets, some differences are unavoidable.
```

Comment:
Is there a reason for the shorter-than-usual line length here?

Reply:
No there is not. I changed it to have the normal line length.